### PR TITLE
Merging to feature abstractions

### PR DESCRIPTION
**Pull request by changhsinlee**
I think I messed up merging features abstractions + develop + my own branch, and I can't automatically merge between develop and my branch.

Tested the idea of a simple stacked model.

### CODE REVIEW

**Comment by changhsinlee** [comment]
@crodriguez1a if you could merge this into develop please show me how to 😭I saw 15 file merge conflicts...

**Comment by crodriguez1a** [comment]
@changhsinlee I made quite a few more commits.  You'll want to re-base and make sure I haven't broken anything for you. Tests were passing, so you should be fine.

**Review by crodriguez1a — CHANGES_REQUESTED** [review]
Pull down the latest commits and re-validate.

File: notebooks/chang/1.4-logisitic-regression-on-all-files.ipynb [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+      "\u001b[1;31mMemoryError\u001b[0m: "
+     ]
+    }
+   ],
+   "source": [
+    "for tag_num, tag_name in enumerate(tags):\n",
+    "    train_files = helpers.find_paths_with_tags(csv_path=TRAIN_CSV, files_path=TRAIN_FILES, filters=[tag_name])\n",
+    "    wav_data = helpers.load_wav_files(train_files, duration=15)\n",
+    "    with open('data/raw/train/wav-data-{}.pkl'.format(tag_name), 'wb') as f:\n",
+    "        pickle.dump(wav_data, f)\n",
```
Comment (crodriguez1a) [inline notebooks/chang/1.4-logisitic-regression-on-all-files.ipynb:138]: @changhsinlee 👍

File: notebooks/chang/1.4-logisitic-regression-on-all-files.ipynb [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+      "Processed 2-Violin_or_fiddle\n",
+      "Processed 3-Finger_snapping\n"
+     ]
+    }
+   ],
+   "source": [
+    "for tag_num, tag_name in enumerate(tags[37:]):\n",
+    "    train_files = helpers.find_paths_with_tags(csv_path=TRAIN_CSV, files_path=TRAIN_FILES, filters=[tag_name])\n",
+    "    wav_data = helpers.load_wav_files(train_files, duration=15)\n",
+    "    with open('data/raw/train/wav-data-{}.pkl'.format(tag_name), 'wb') as f:\n",
```
Comment (crodriguez1a) [inline notebooks/chang/1.4-logisitic-regression-on-all-files.ipynb:162]: @changhsinlee As of 3.6, you can use string literals to concatenate 

```python
with open(f'data/raw/train/wav-data-{tag_name}.pkl', 'wb') as f:
```

**Comment by crodriguez1a** [comment]
Closing while we sort out commit history

**Pull request closed without merging**